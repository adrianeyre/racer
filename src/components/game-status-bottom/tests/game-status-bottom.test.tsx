import React from 'react';
import { shallow } from 'enzyme';

import GameStatusBottom from '../game-status-bottom';
import IGameStatusBottomProps from '../interfaces/game-status-bottom-props';

describe('Game Status Bottom', () => {
	it('Should render correctly', () => {
		const defaultProps: IGameStatusBottomProps = {
			lives: 1000,
			time: 9999,
		};

		const gameStatus = shallow(<GameStatusBottom {...defaultProps} />);
		expect(gameStatus).toMatchSnapshot();
	});
});