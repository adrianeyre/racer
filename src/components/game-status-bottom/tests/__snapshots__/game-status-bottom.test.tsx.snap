// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game Status Bottom Should render correctly 1`] = `ShallowWrapper {}`;
